src/bulk.rs

```diff
@@ -28,18 +29,21 @@ struct Parser {
     duration: HistogramVec,
     block_count: CounterVec,
     bytes_read: Histogram,
+    config: Config,
 }
 
 impl Parser {
     fn new(
         daemon: &Daemon,
         metrics: &Metrics,
         indexed_blockhashes: HashSet<Sha256dHash>,
+        config: &Config,
     ) -> Result<Arc<Parser>> {
         Ok(Arc::new(Parser {
             magic: daemon.magic(),
             current_headers: load_headers(daemon)?,
             indexed_blockhashes: Mutex::new(indexed_blockhashes),
+            config: config.clone(), // @fixme
```

Comment:
Maybe we can use an Arc here and in other places (to have one instance of the configuration)?

Reply:
Yeah, I think we definitely should. Do you think we should use an Arc for the config, or pass around the (already Arc-ed) `App` instead?

Reply:
I prefer using Arc for the config (since I prefer not to create too much dependencies over the App object - as it is becoming a ["God object"](https://en.wikipedia.org/wiki/God_object)).

Reply:
We ended up saving just `extended_db_enabled` instead of the whole `Config`, which made this moot.